sparql/absences.sparql

```diff
@@ -1,4 +1,4 @@
-@prefix: http://www.w3.org/2000/01/rdf-schema#
+@prefix rdfs: http://www.w3.org/2000/01/rdf-schema#
```

Comment:
In SPARQL I don't think there can be an `@` (but this works for Turtle). Use `PREFIX` instead.